db/migrate/20160219142640_add_caren_bishop_to_licensees.rb

```diff
@@ -0,0 +1,12 @@
+class AddCarenBishopToLicensees < ActiveRecord::Migration
+  def change
+    add_column :licensees, :surname, :string
+    add_column :licensees, :given_name, :string
+    add_column :licensees, :gender, :string
+    add_column :licensees, :height, :integer
+    add_column :licensees, :weight, :integer
+    add_column :licensees, :born_on, :date
+    add_column :licensees, :eye_color, :string
+    add_column :licensees, :hair_color, :string
+  end
+end
```

Comment:
Sorry, you're barking up the wrong tree with this one. Migrations are for making structural changes to the database. To make a content change, like adding a new row, there's no need to create a migration - just call `Licensee.create`